templates/components/form/buttons.html.twig

```diff
@@ -156,9 +156,5 @@
       </div>
 
       <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}" />
-   </div> {# #mainformtable #}
-</form> {# [name=asset_form] #}
-    {% else %}
-   </div> {# #mainformtable #}
```

Comment:
These are opened in the `templates/components/form/header.html.twig` template that is included prior to the `templates/components/form/header.html.twig` template in our forms.
I do not like the idea of opening tags in a template and close them in another template, this is prone to errors.